### DISCUSSION TITLE
Izing

### DISCUSSION THREAD

**Discussion by botusist** [community/community#62487] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Quanto fica a versão premium do izing e o que ela tem? esta funcionando por completo e ela é saas?

**Comment by queenofcorgis** [comment]
Hi @botusist , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

**Reply by Nescaubomba** [reply]
How much is the premium version of izing and what does it have? Is it working completely and is it safe?
